wishlistApp/views.py

```diff
@@ -75,8 +75,28 @@ def delete_account(request):
 
 @login_required
 def user_list(request):
-    return render(request, 'wishlistApp/userList.html')
+    user = request.user.get_username()
+    return render(request, 'wishlistApp/userList.html', {'user':user})
```

Comment:
Currently your (username)'s List isn't showing up. I think this is because you're passing in a string in `'user'`, but when you try to get the variable in your userList.html, you're using `{{ user.get_username }}`. Getting variables is like getting API data, and it's looking for a sub variable/category of "get_username" which doesnt exist.
The way I did it in profile.html/profile view was to just get `{{ user.username }}` in profile.html without passing anything in through the view, but I think you can try changing your userList.html to have `{{ user }}` first